ReactiveUI/Activation.cs

```diff
@@ -37,9 +36,12 @@ namespace ReactiveUI
     public sealed class ViewModelActivator
     {
         readonly List<Func<IEnumerable<IDisposable>>> blocks;
+        readonly Subject<Unit> deactivated = new Subject<Unit>();
```

Comment:
For consistency, can we move object creation to the ctor, since that is where `blocks` is instantiated?